fili-generic-example/README.md

```diff
@@ -13,6 +13,15 @@ Note that this was last tested using [version 0.9.1](https://github.com/yahoo/fi
 
 ## Setup and Launching
 
+### Docker
+There is a Fili Dockerfile which is available on Dockerhub. To start playing around with Fili, you can download and open Docker. Then
+run the command
+```bash
+docker run --name fili-generic-example -i --rm -p 3001:8081 -p 3000:8082 fili:1.0
```

Comment:
Thanks for the suggestions. I have added those commands into the readme. I also thought that the 9998 mapping was necessary, but when I tried to run with ``` -p 9998:9998 ``` Docker gave me an error:

```
Error response from daemon: driver failed programming external connectivity on endpoint fili-wikipedia-example
Error starting userland proxy: Bind for 0.0.0.0:9998 failed: port is already allocated.
```